installscripts/cookbooks/jenkins/recipes/prereqs.rb

```diff
@@ -13,3 +13,7 @@
 end
 
 npm_package 'jshint'
+
+npm_package 'protractor'
```

Comment:
Why are we adding UI automation utilities to jazz's installer? This is required on the host machine that will run the UI tests which is not the same as the Jenkins that Jazz uses.